fabric-biomes-v1/src/main/java/net/fabricmc/fabric/api/biomes/v1/OverworldBiomes.java

```diff
@@ -107,4 +107,29 @@ public static void addBiomeVariant(Biome replaced, Biome variant, double chance,
 	public static void setRiverBiome(Biome parent, Biome river) {
 		InternalBiomeData.setOverworldRiverBiome(parent, river);
 	}
+
+	/**
+	 * Adds the biome as a large edge biome of the parent biome, with the specified weight.
```

Comment:
Should clarify how large these large edges are -- they're about the size of hills, or are they a bit smaller?

Reply:
They're at the same scale as hills and sub biomes